be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/ApiControllers/IdentityApiController.cs

```diff
@@ -114,6 +121,17 @@ public async Task<IActionResult> Index([FromBody] SetUserPasswordRequestModel mo
                     {
                         await _userService.SignInAsync(user.Email, model.Password, null, null);
 
+                        if (model.ClientApprovals.Any())
```

Comment:
**suggestion:** Add check if model.ClientApprovals are not null. If ClienApprovals would be null you receive null exception